backend/src/services/objective.service.ts

```diff
@@ -0,0 +1,74 @@
+import { getCustomRepository } from 'typeorm';
+import { Objective } from '~/data/entities/objective';
+import { OKR } from '~/data/entities/okr';
+import Okrepository from '~/data/repositories/okr.repository';
+import ObjectiveRepository from '~/data/repositories/objective.repository';
+import { badRequestError } from '~/common/errors';
+
+export const createObjectiveToOkr = async ({
+  okrId,
+  body,
+}: {
+  okrId: string;
+  body: Objective;
+}): Promise<OKR> => {
+  if (!body.name) throw badRequestError('Objective name is undefined!!!');
```

Comment:
Since you use `validateBody` middleware you do not need these validations anymore